### PR TITLE
Additional changes

### PR DESCRIPTION
**Pull request by bitwiseman**
Abstract made really abstract.
Remove the rest of the you pronouns.
Added process overview place holder.

### CODE REVIEW

**Comment by bitwiseman** [comment]
WORK IN PROGRESS - The Overview section is obviously not complete.

File: jep/1/README.adoc [hunk 1]
```diff
@@ -312,6 +335,8 @@ are never meant to be completed. E.g. JEP 1 (this JEP).
 
 ==== JEP Maintenance
 
+After a JEP reaches Approved it may or may not require
```
Comment (rtyler) [inline jep/1/README.adoc]: Don't forget this dangling sentence

File: jep/1/README.adoc [hunk 2]
```diff
@@ -138,6 +151,17 @@ below:
 
 === JEP Workflow
 
+==== Overview 
+
+The JEP process has a number of stages
+
+. **Preparation** - one or two sentences about this
```
Comment (rtyler) [inline jep/1/README.adoc]: Seems like a reasonable overview structure to start with to me

**Comment by bitwiseman** [comment]
@rtyler - Okay, I'm going to call this a stopping point.  Please review and I'll address feedback.

**Pull request merged**